src/references/audio.md

```diff
@@ -52,29 +48,31 @@ Enables spatialization of a sound. This will allow your sound to be heard in spe
 
 Enables volume based on distance of a sound. This essentially means the sound will fade out to 0 volume at a distance and vice-versa.
 
-To achieve full 3D positional sound, ensure that you enable both spatialization and attenuation.
+To achieve full 3D positional sound, ensure that both spatialization and attenuation are enabled.
 
 Background ambience and music is best suited to have spatialization and attenuation disabled.
 
-For example, a simple way to do "audio zones" an area on your map you want to hear certain music, you can disable spatialization and enable attenuation to easily fade in a music track that will be heard in stereo (left and right speakers) where you want players to hear it. You'll have to adjust radius and falloff accordingly as needed.
+For example, a simple way to do "audio zones" in an area on a map to hear certain music, disable spatialization and enable attenuation to easily fade in a music track that will be heard in stereo (left and right speakers) where creators want their players to hear it. Creators will have to adjust radius and falloff accordingly as needed.
 
 #### Enable Occlusion
 
 When enabled the audio will change attenuation if there is geometry between the player and the audio. The audio engine will handle volume attenuation, low pass frequency filters and other digital signal processing effects (DSP) in real time accordingly when geometry occludes the sound source and the listener. Most sounds that exist in 3D space in your game world should have this option enabled.
 
 #### Auto Play
 
-When enabled the sound will automatically upon object spawn.
+Causes the audio to player immediately upon additional to the world.
```

Comment:
Weird wording. Causes the audio to play~er~ immediately upon addition~al~ to the world.